explainer.md

```diff
@@ -187,6 +187,30 @@ There are several open questions to address:
    source.
 1. How to handle messaging.
 
+## Security and privacy considerations
+
+See [the corresponding section](
+https://w3c.github.io/presentation-api/#security-and-privacy-considerations) of
+the Presentation API spec. Below, we list characteristics unique to site
+initiated mirroring.
```

Comment:
There are also some aspects that do not apply to site-initiate mirroring, like the discussion of DIAL URLs and connection from multiple controllers.  It would be worth mentioning what aspects of the generic API usage don't apply here.

Reply:
Added mentions of DIAL, private browsing, and messaging.
I don't know if there's a need to specify that reconnect() from multiple controllers should be disallowed. Reconnecting to a mirroring session would just be uninteresting as mentioned below. I didn't find any mentions of reconnect() not being possible for 1UA in the spec.